docs/workflow/continuous_integration.rst

```diff
@@ -0,0 +1,181 @@
+******************
+Continuous Integration
+******************
+
+We use a so-called [Continuous Integration](http://en.wikipedia.org/wiki/Continuous_integration) 
+workflow with TARDIS. This means that each time a change is proposed (via pull-request) 
+or a proposed change is merged into the main master branch a service will download the version 
+and execute all the unit tests and integration tests offered with TARDIS. This helps 
+us detect bugs immediately. 
+
+The following pages explain how we setup automated debugging on TARDIS through a remote 
+cloud service, called azure, hosted by visual studio team services. This is done by 
```

Comment:
This content is great! It will appear even more better if you can capitalize the 1st letter of each word in product names like: `azure`, `visual studio`, etc.